### PR TITLE
JMS Workshop1 FY2023 Q1 Updates

### PR DESCRIPTION
**Pull request by ESoapW**
[Reference](https://confluence.oci.oraclecorp.com/display/AJ/FY2023+Quarterly+Review+and+Enhancements)

### CODE REVIEW

File: java-management/set-up-of-management-agent-linux/set-up-of-management-agent-linux.md [hunk 1]
```diff
@@ -188,43 +188,29 @@ Install Management Agent (If your host is Windows, skip to Lab 6: Install Manage
...
-10. Select the compartment that the fleet is in and click the fleet.
-
-11. Click on **Modify Agent Settings**.
-
-  ![image of fleet details page](/../images/fleet-details-page-new.png)
-
-12. Change the **Java Runtime Discovery** and **Java Runtime Usage** to the desired value. For this example, change **Java Runtime Discovery** to **3 hours**, and **Java Runtime Usage** to **5 minutes**.
-
-  ![image of modify agent settings page](/../images/fleet-modify-agent-settings-new.png)
+8. JMS has been linked to the management agent and will collect information on your Java runtimes. The scanning frequency was set in [Lab 2: Set Up a Fleet](?lab=setup-a-fleet).
```
Comment (tchoo-oracle) [inline java-management/set-up-of-management-agent-linux/set-up-of-management-agent-linux.md:213]: JMS has been linked to the management agent and will collect information on your Java runtimes based on the scanning frequency defined in Lab 2

File: java-management/set-up-of-management-agent-windows/set-up-of-management-agent-windows.md [hunk 2]
```diff
@@ -272,45 +272,31 @@ Install Management Agent (If your host is Linux, skip to skip to Lab 5: Install
...
-10. Select the compartment that the fleet is in and click the fleet.
-
-11. Click on **Modify Agent Settings**.
-
-  ![image of fleet details page](/../images/fleet-details-page-new.png)
-
-12. Change the **Java Runtime Discovery** and **Java Runtime Usage** to the desired value. For this example, change **Java Runtime Discovery** to **3 hours**, and **Java Runtime Usage** to **5 minutes**.
-
-  ![image of modify agent settings page](/../images/fleet-modify-agent-settings-new.png)
+8. JMS has been linked to the management agent and will collect information on your Java runtimes. The scanning frequency was set in [Lab 2: Set Up a Fleet](?lab=setup-a-fleet).
```
Comment (tchoo-oracle) [inline java-management/set-up-of-management-agent-windows/set-up-of-management-agent-windows.md:299]: same comment as above

**Review by tchoo-oracle — CHANGES_REQUESTED** [review]
pls resolve feedback and update PR

File: java-management/deploy-a-java-application/deploy-a-java-application.md [hunk 3]
```diff
@@ -242,6 +242,13 @@ You may now **proceed to the next lab.**
     </copy>
     ```
 
+**For Task 3**
+
+* If you encounter an error similar to the following:
+    ```
+    No match for argument: jdk1.8.x86_64
+    ```
+  Please use a new instance with AMD or Intel shape.
```
Comment (enhl-orc) [inline java-management/deploy-a-java-application/deploy-a-java-application.md:251]: Please use a new instance with AMD or Intel shape instead of Ampere.

**Review by enhl-orc — CHANGES_REQUESTED** [review]
Left minor comments.

**Review by tchoo-oracle — APPROVED** [review]

**Review by enhl-orc — APPROVED** [review]

**Pull request merged**